src/components/MultipleAvatars.tsx

```diff
@@ -93,8 +78,31 @@ function MultipleAvatars({
     shouldUseCardBackground = false,
     maxAvatarsInRow = CONST.AVATAR_ROW_SIZE.DEFAULT,
 }: MultipleAvatarsProps) {
+    const theme = useTheme();
+    const styles = useThemeStyles();
+
+    const avatarSizeToStylesMap: AvatarSizeToStylesMap = useMemo(
+        () => ({
+            [CONST.AVATAR_SIZE.SMALL]: {
+                singleAvatarStyle: styles.singleAvatarSmall,
+                secondAvatarStyles: styles.secondAvatarSmall,
+            },
+            [CONST.AVATAR_SIZE.LARGE]: {
+                singleAvatarStyle: styles.singleAvatarMedium,
+                secondAvatarStyles: styles.secondAvatarMedium,
+            },
+            [CONST.AVATAR_SIZE.DEFAULT]: {
+                singleAvatarStyle: styles.singleAvatar,
+                secondAvatarStyles: styles.secondAvatar,
+            },
+        }),
+        [styles],
+    );
+
+    const secondAvatarDefaultStyles = secondAvatarStyle ?? [StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)];
```

Comment:
I think we should rename this variable to `secondAvatarStyle` and rename the prop `secondAvatarStyle` to `secondAvatarStyleProp`.

Especially i think this should be singular instead of plural, since the prop is also singular. (sorry for nitpicking 😃)

Reply:
But it would break the convention we have in the app - all the styles passed as props doesn't have 'props' in the name. `badgeStyles`, `textStyles`, `containerStyles` or even `listContainerStyles` in BaseOptionsSelector below - these are just some of examples.

Reply:
which convention do you mean?

Reply:
Naming the props. I just gave some examples how the style props are named. We don't have containerStylesProp only containerStyles. If I changed secondAvatarStyle to secondAvatarStyleProp that would be the only style prop in the app which name includes 'Prop'

Reply:
I don't think we need to add `Prop` to the variable name, but if `secondAvatarStyle` can be an array of styles, maybe we rename to `secondAvatarStyles`?

Reply:
@grgia yeah, that makes sense. But can we do this as a follow-up? This prop is passed from 6 different components and the name in plural would collide with another const in the file - https://github.com/Expensify/App/blob/e909316f451a08aedd420c31f19cc3ea3cac0243/src/components/MultipleAvatars.tsx#L97
(it was introduced earlier, not in this PR, so I guess that the prop was deliberately in singular)
wdyt?

Reply:
I think it would be safer to work on the names refactor in a separate PR 🙂

Reply:
oh sorry, i might not have explained this well...

i meant to create an alias for the prop just in this component. Like this:

```ts
function MultipleAvatars({
     ...
     secondAvatarStyle: secondAvatarStyleProp,
     ...
}) {
...
```

Reply:
Yes I agree, no need for name refactor here

Reply:
ah, ok then 😃 thanks for clarification

Reply:
@chrispader changes applied 🙂